src/review/models.py

```diff
@@ -306,16 +307,27 @@ class ReviewForm(models.Model):
     )
 
     name = models.CharField(max_length=200)
-    slug = models.SlugField(max_length=200)
+    slug = models.SlugField(
+        max_length=200,
+        blank=True,
+    )
 
-    intro = models.TextField(help_text="Message displayed at the start of the review form.")
-    thanks = models.TextField(help_text="Message displayed after the reviewer is finished.")
+    intro = models.TextField(
+        help_text="Message displayed at the start of the review form.",
+    )
+    thanks = models.TextField(
+        help_text="Message displayed after the reviewer is finished.",
+    )
 
     elements = models.ManyToManyField('ReviewFormElement')
     deleted = models.BooleanField(default=False)
 
+    def save(self, *args, **kwargs):
+        self.slug = slugify(self.name)
```

Comment:
This would make the `slug` change when the name changes, do you think it might be a problem? (e.g. if we use it for URLs that might then 404)

I had a quick look and I don't think we use the slug at all for review forms. If that is the case, I'd advocate for removing it entirely

Reply:
I considered that. The only time we use this is to identify if there is or isn’t a default-form so it’s worth leaving it in.

Reply:
What about:
```py
def save(self, *args, **kwargs):
    if not self.slug:
        self.slug = slugify(self.name)
```

Reply:
> 

Where do we do this?

Reply:
https://github.com/search?q=repo%3ABirkbeckCTP%2Fimports%20default-form&type=code

Reply:
It looks like it's checked here:
https://github.com/BirkbeckCTP/janeway/blob/c4a0ada031820fdd0a4a9904d22bb4fed1773445/src/journal/models.py#L1254-L1261

Reply:
Why do you think it should update @ajrbyers? I'm not familiar with what you have in mind, because I don't know what the slug is for in this case. Why do we have it?

I'm also in favor of removing it, if that makes things simpler.

Reply:
The slug field is intended to be a slug representation of the name field, if you change the name the slug should also change. As I said, we can remove it, but that's not what this PR is about and would require work on other repositories.